ParisBNF/abb/BNFabb57.xml

```diff
@@ -276,31 +277,30 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
 
                 <item xml:id="a8">
                   <locus target="#31va"/>
-                  <desc type="PoemQene">A <term key="Qene">qǝne-poem</term>, possibly by Lāʾǝka Māryām</desc>
-                  <q xml:lang="gez">ሊቀ፡ ካህናት፡ አሮን፡ በደመ፡ ላህም፡ እንስሳ፡ ለደብተራሁ፡ ቀደሳ፡ በከመ፡ ህጎሙ፡ ቀዳሚት፡ ከመ፡᎓
+                  <desc type="GuestText"><title type="complete" ref="LIT6963Laseqlata">A <term key="Qene">qǝne-poem</term>, possibly by Lāʾǝka Māryām</title></desc>  
```

Comment:
mark up inside of the title element often causes problems, better close the title first 
`<title type="complete" ref="LIT6963Laseqlata"/>A <term key="Qene">qǝne-poem</term>, possibly by Lāʾǝka Māryām</desc>`